### PR TITLE
DE43090 Add translations for the Applied Filters component

### PR DESCRIPTION
**Pull request by jadasc**
## Description
* Added new `.js` files for each language 
## Testing Steps
To test in Portfolio:
1. Open the [folio-app](https://github.com/Brightspace/folio-app) project
2. Either:
    i. From the terminal, run 
```npm install git://github.com/BrightspaceUILabs/facet-filter-sort.git#ec60c9c25feab7f70d97bb20a8ca7bc69e4d7907```
    OR
    ii. Update `package.json` to:
    ```
    "dependencies": {
        "@brightspace-ui-labs/facet-filter-sort": "BrightspaceUILabs/facet-filter-sort#ec60c9c25feab7f70d97bb20a8ca7bc69e4d7907",
    ```
3. Run Portfolio locally
4. Log in as an instructor
5. Change the user's default language so that it's not English
6. Navigate to Portfolio
7. Toggle to the Evidence View
8. Verify that the "Applied Filters" section is translated according to the language selected in step 5
## Rally
https://rally1.rallydev.com/#/378960033600d/dashboard?detail=%2Fdefect%2F518698053064&fdp=true?fdp=true

### CODE REVIEW

File: components/applied-filters/applied-filters.js [hunk 1]
```diff
@@ -7,6 +7,23 @@ import { getComposedChildren } from '@brightspace-ui/core/helpers/dom';
...
+// import ja from './lang/ja.js';
+import ko from './lang/ko.js';
+import nl from './lang/nl.js';
+import ptBr from './lang/pt-br.js';
+import sv from './lang/sv.js';
+import tr from './lang/tr.js';
+// import zh from  './lang/zh.js';
+import zhCn from  './lang/zh-CN.js';
+import zhTw from './lang/zh-TW.js';
+/* eslint sort-imports: "warn" */
```
Comment (jadasc) [inline components/applied-filters/applied-filters.js:25]: Imports are commented out as we are still awaiting translations

File: components/applied-filters/applied-filters.js [hunk 2]
```diff
@@ -7,6 +7,23 @@ import { getComposedChildren } from '@brightspace-ui/core/helpers/dom';
 import { LocalizeStaticMixin } from '@brightspace-ui/core/mixins/localize-static-mixin.js';
 import { RtlMixin } from '@brightspace-ui/core/mixins/rtl-mixin.js';
 
+/* eslint sort-imports: "off" */
+import arSa from './lang/ar-sa.js';
+// import de from './lang/de.js';
+import en from './lang/en.js';
+// import es from './lang/es.js';
+import frCa from './lang/fr-ca.js';
```
Comment (jameswklassen) [inline components/applied-filters/applied-filters.js:15]: Is this component going to have fr-ON translations as well?

Comment (jadasc) [reply components/applied-filters/applied-filters.js:15]: I had that question as well. Jeff said that this component doesn't require fr-ON translations 👍

**Review by EvanLemieux — APPROVED** [review]
LGTM 👍

**Comment by jadasc** [comment]
Do I have to bump the patch version at all?

**Comment by jstefaniuk-d2l** [comment]
https://github.com/BrightspaceUILabs/facet-filter-sort#version-changes

**Review by jameswklassen — APPROVED** [review]

**Pull request merged**